text/0000-async-project-goal-2024h2.md

```diff
@@ -0,0 +1,275 @@
+| ↳ author RFC                             | [nikomatsakis][] or [compiler-errors][] | ![Funded][]         |
+| ↳ approve RFC                            | ![Team][] [Lang]                        |                     |
+| ↳ stabilization                          | [compiler-errors][]                     | ![Funded][]         |
+| resolve the ["send bound"][sb] problem   |                                         | ![Funded][]         |
+| ↳ ~~RTN implementation~~                 | ~~[compiler-errors][]~~                 | ![Complete][]       |
+| ↳ ~~RTN RFC~~                            | [nikomatsakis][]                        | ![Complete][]       |
+| ↳ approve RTN RFC or provide alternative | ![Team][] [Lang]                        |                     |
+| ↳ stabilization                          | [compiler-errors][]                     | ![Funded][]         |
+| stabilize trait for async iteration      |                                         | ![Funded][]         |
+| ↳ author RFC                             | [eholk][]                               | ![Funded][]         |
+| ↳ approve RFC                            | ![Team][] [Libs-API]                    | ![Funded][]         |
+| ↳ implementation                         | [eholk][]                               | ![Funded][]         |
+| author draft RFC for async vision        |                                         | ![Funded][]         |
+| ↳ author RFC                             | [tmandry][]                             | ![Funded][]         |
+| ↳ approve RFC                            | ![Team][] [Lang], [Libs-API]            |                     |
+| complete async drop experiments          |                                         |                     |
+| ↳ ~~author MCP~~                         | ~~[petrochenkov][]~~                    | ![Complete][]       |
+| ↳ ~~approve MCP~~                        | ~~[Compiler]~~                          | ![Complete][]       |
+| ↳ implementation work                    | [petrochenkov][]                        | ![Not funded][] (*) |
+
+(*) Implementation work on async drop experiments is currently unfunded. We are trying to figure out next steps.
+
+[Funded]: https://img.shields.io/badge/Funded-yellow
+[Not funded]: https://img.shields.io/badge/Not%20yet%20funded-red
+[Approved]: https://img.shields.io/badge/Approved-green
+[Not approved]: https://img.shields.io/badge/Not%20yet%20approved-red
+[Complete]: https://img.shields.io/badge/Complete-green
+[TBD]: https://img.shields.io/badge/TBD-red
+
+
+## Support needed from the project
+
+Agreement from [Lang] and [Libs-API] to the items marked ![Team][] in the table above. Potentially other design meetings as needed.
+
+Expectation is that 3-4 design meetings will be needed from lang over the course of H2 and 1-2 from libs API. Reviewing the async vision doc is expected to be the biggest requirement.
+
+# Outputs and milestones
```

Comment:
This section seems incomplete.. but also a bit redundant.

Reply:
Yes, it should just be removed.